### PR TITLE
chore: integrating currents reporter tool

### PR DESCRIPTION
**Pull request by ondracja**
- adding project connection to Currents tool to our E2E desktop tests

### CODE REVIEW

File: packages/suite-desktop-core/package.json [hunk 1]
```diff
@@ -47,6 +48,7 @@
         "@trezor/trezor-user-env-link": "workspace:*",
         "@trezor/type-utils": "workspace:*",
         "@types/electron-localshortcut": "^3.1.3",
+        "dotenv": "^16.4.5",
```
Comment (mroz22) [inline packages/suite-desktop-core/package.json]: is it really needed?

File: .github/workflows/test-suite-desktop-e2e.yml [hunk 2]
```diff
@@ -61,6 +61,7 @@ jobs:
 
       - name: Run e2e desktop tests
         env:
```
Comment (vdovhanych) [inline .github/workflows/test-suite-desktop-e2e.yml:73]: its missing the 

```suggestion
        env:
          CURRENTS_RECORD_KEY: {{ secrets.CURRENTS_RECORD_KEY }}
```

File: packages/suite-desktop-core/package.json (hunk 1 above)
Comment (ondracja) [reply packages/suite-desktop-core/package.json]: It is indeed not needed, I can supply it locally via cli args. Removed.

**Review by vdovhanych — COMMENTED** [review]
can we name it test ony? 

```
name: "[Test] suite-desktop e2e"
```
no need to have build in the name.

**Review by vdovhanych — APPROVED** [review]
Just rename it and lets merge it.

**Pull request merged**